docs/logicapps-ai-course/01-overview.md

```diff
@@ -1,12 +1,55 @@
+---
+title: Overview of building a conversational agent in Azure Logic Apps
+description: An overview of the lesson that helps you build and deploy a conversational agent in Azure Logic Apps.
+ms.service: logic-apps
+ms.topic: tutorial
+ms.date: 08/19/2025
+author: absaafan
+ms.author: absaafan
+---
+
 # Overview
 
-- Conceptual introduction about what is possible and differentiate between what Azure Logic Apps brings.
-- Durable execution, enterprise scale, monitoring, security features, connectivity, and so on, for example, use Azure Logic Apps for agents.
+This lesson introduces what’s possible when you build conversational and autonomous agents using Azure Logic Apps. It explains the unique capabilities that Azure Logic Apps brings to agent architectures and gives a short outline of this course.
+
+## Introduction
+
+Agents combine reasoning, state, and tools to take actions in response to user inputs or events. Using Azure Logic Apps for agents lets you separate the agent’s decision logic (the model, prompts, and orchestration) from reliable, production-grade execution and integration.
+
+Azure Logic Apps provides a visual workflow engine that runs in a cloud service designed for integration and orchestration. Adding agent loop(s) into your workflow unlocks powerful AI reasoning coupled with the extensive features and proven reliability that Azure Logic Apps are known for.
```

Comment:
Azure Logic Apps provides an enterprise grade workflow engine with a visual designer and offers multiple hosting options ranging from on Prem to cloud.